iset.mm

```diff
@@ -29490,13 +29507,28 @@ intersection or the difference (that is, this theorem would be equality
     $d x A $.
     $( Theorem 19.2 of [Margaris] p. 89 with restricted quantifiers (compare
        ~ 19.2 ).  The restricted version is valid only when the domain of
-       quantification is inhabited.  (Contributed by Jim Kingdon,
-       5-Aug-2018.) $)
-    r19.2m $p |- ( ( E. x x e. A /\ A. x e. A ph ) -> E. x e. A ph ) $=
+       quantification is inhabited.  (Contributed by Jim Kingdon, 5-Aug-2018.)
+       New proofs should use ~ r19.2m which is the same but which allows ` x `
+       and ` y ` to be either the same variable or different ones.
+       (Proof modification is discouraged.)  (New usage is discouraged.) $)
+    r19.2mOLD $p |- ( ( E. x x e. A /\ A. x e. A ph ) -> E. x e. A ph ) $=
```

Comment:
This may be called r19.2mALT - OLD theorems will/should be removed after one year...

Reply:
Well it is an OLD in the sense that the plan is to remove it (yes, in favor of the new r19.2m). I don't think ALT would make sense because isn't that for things which have some reason to continue to exist?

As far as I can tell the best plan is to stick with the OLD nomenclature but follow more of the conventions at https://us.metamath.org/mpeuni/conventions.html . I've pushed up a change to do that. I suppose if we are really committed to exactly a year (rather than at least a year) we can always make sure to switch over the usages before then (it is only 11).

Reply:
I do not now the procedure for iset.mm, but for set.mm, several people check if there are obsolete theorems older than 1 year and delete them. This causes trouble if they are still in use. therefore, their usage should be eliminated as soon as possible.

Reply:
The most obvious difference in iset.mm is that we've been deleting theorems immediately if we don't want them and there are no usages. I suppose if there are more people working on it that might change some day, but at least speaking for myself my preference is to continue with the immediate deletions unless people report this is causing problems.